IMG_data_items.tex

```diff
@@ -882,319 +883,367 @@ \subsubsection{Master calibration / intermediate data item structures}\label{sss
+  Description: & LM band image with flux calibration, WC coordinate system and distorion information\\[0.3cm]
+  \FITS{PRO.CATG}: & \CODE{LM_SCI_CALIBRATED}\\[0.3cm]
+  OCA keywords: & \FITS{PRO.CATG}, \FITS{DRS.FILTER}\\
+  \FITS{DO.CATG}: & \CODE{LM_SCI_CALIBRATED}\\[0.3cm]
+  % Template: & \TPL{METIS_img_lm_cal_standard}\\
+  % & \TPL{METIS_img_lm_*_obs_*}       \\
+  Created by:   & \REC{metis_lm_img_calibrate} \\
+  Input for:    & \REC{metis_img_adi_cgrph} \\
+                & \REC{metis_lm_adi_app} \\
+                & \REC{metis_lm_img_sci_postprocess} \\
+  Processing \ac{FITS} Keywords: & provided at \ac{PAE}\\
+  % Data item structure: & see \ref{drsstructure:lmrsrfraw}\\
 \end{recipedef}
-%\paragraph{\PROD{LM_SCI_CALIBRATED}}\label{drsstructure:LM_SCI_CALIBRATED}
+% \paragraph{\PROD{LM_SCI_CALIBRATED}}\label{drsstructure:LM_SCI_CALIBRATED}
 \begin{datastructdef}
-\textbf{Corresponding \ac{CPL} structure:}
-\begin{enumerate}
-    \item \texttt{cpl\_propertylist * keywords: Primary keywords (\FITS{DPR.CATG},  \FITS{DPR.TECH},  \FITS{DPR.TYPE},  \FITS{INS.OPTI3.NAME},  \FITS{INS.OPTI9.NAME},  \FITS{INS.OPTI10.NAME}, \FITS{DRS.FILTER})}
-    \item \texttt{hdrl\_image * image: Single image}
-    \item \texttt{cpl\_propertylist * plistarray[]: Extension keywords}
-\end{enumerate}
-\end{datastructdef}    
+  \textbf{Corresponding \ac{CPL} structure:}
+  \begin{enumerate}
+  \item \texttt{cpl\_propertylist * keywords: Primary keywords (\FITS{DPR.CATG},  \FITS{DPR.TECH},  \FITS{DPR.TYPE},  \FITS{INS.OPTI3.NAME},  \FITS{INS.OPTI9.NAME},  \FITS{INS.OPTI10.NAME}, \FITS{DRS.FILTER})}
+  \item \texttt{hdrl\_image * image: Single image}
+  \item \texttt{cpl\_propertylist * plistarray[]: Extension keywords}
+  \end{enumerate}
+\end{datastructdef}
 
 
 \paragraph{\PROD{N_SCI_CALIBRATED}}\label{dataitem:n_sci_calibrated}
 \begin{recipedef}
-\textbf{\ac{FITS} file structure:}\\
-Name: & \PROD{N_SCI_CALIBRATED}\\[0.3cm]
-Description: & N band image with flux calibration and distorion information\\[0.3cm]
-\FITS{PRO.CATG}: & \CODE{N_SCI_CALIBRATED} \\[0.3cm]
-OCA keywords: & \FITS{PRO.CATG}, \FITS{DRS.FILTER}\\
-\FITS{DO.CATG}: & \CODE{N_SCI_CALIBRATED}\\[0.3cm]
-% Template: & \TPL{METIS_img_n_cal_standard}\\
-%             & \TPL{METIS_img_n_*_obs_*}       \\
-Created by:   & \REC{metis_n_img_calibrate} \\
-Input for:    & \REC{metis_img_adi_cgrph} \\
-              & \REC{metis_n_img_restore} \\
-Processing \ac{FITS} Keywords: & provided at \ac{PAE}\\
-%Data item structure: & see \ref{drsstructure:nrsrfraw}\\
+  \textbf{\ac{FITS} file structure:}\\
+  Name: & \PROD{N_SCI_CALIBRATED}\\[0.3cm]
+  Description: & N band image with flux calibration and distorion information\\[0.3cm]
+  \FITS{PRO.CATG}: & \CODE{N_SCI_CALIBRATED} \\[0.3cm]
+  OCA keywords: & \FITS{PRO.CATG}, \FITS{DRS.FILTER}\\
+  \FITS{DO.CATG}: & \CODE{N_SCI_CALIBRATED}\\[0.3cm]
+  % Template: & \TPL{METIS_img_n_cal_standard}\\
+  % & \TPL{METIS_img_n_*_obs_*}       \\
```

Comment:
Found one case where your auto formatter made things worse! But overall it is for the better.